.github/workflows/lint-and-test.yml

```diff
@@ -44,26 +44,22 @@ jobs:
           architecture: x64
 
       - uses: DeLaGuardo/setup-clojure@master
-        with:
-          lein: '2.11.2'
 
       - name: Show versions
         run: |
           java -version
-          lein version
 
       - name: Cache dependencies
         uses: actions/cache@v4
         with:
           path: |
             ~/.m2
-            ~/.lein
           key: clj-cache-test-${{ hashFiles('project.clj') }}
           restore-keys: |
             clj-cache-test-
 
       - name: Run tests
-        run: lein test-integration
+        run: make test-integration
 
       - name: Run cloverage
```

Comment:
This workflow uses `lein` command yet.
Could you delete or fix like below?
https://github.com/liquidz/antq/blob/main/.github/workflows/coverage.yml

Reply:
Resolved at 82597bb